source/initial-dns-seedlist-discovery/tests/README.rst

```diff
@@ -23,11 +23,17 @@ these tests::
   _mongodb._tcp.test3.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
   _mongodb._tcp.test5.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
   _mongodb._tcp.test6.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
+  _mongodb._tcp.test7.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
+  _mongodb._tcp.test8.test.build.10gen.cc.  86400  IN SRV  27017  localhost.build.10gen.cc.
 
   Record                                    TTL    Class   Text
   test5.test.build.10gen.cc.                86400  IN TXT  "connectTimeoutMS=300000&socketTimeoutMS=300000"
   test6.test.build.10gen.cc.                86400  IN TXT  "connectTimeoutMS=200000"
   test6.test.build.10gen.cc.                86400  IN TXT  "socketTimeoutMS=200000"
+  test7.test.build.10gen.cc.                86400  IN TXT  "readPreference=secondaryPreferred&readPreferenceTags=🥃"
```

Comment:
Brilliant.

Reply:
Picked it over a normal smiley, as a smiley is part of the BMP, but this glass is not.